packages/beeq/src/components/alert/scss/bq-alert.scss

```
@@ -0,0 +1,83 @@
+/* -------------------------------------------------------------------------- */
+/*                        Alert styles                        */
+/* -------------------------------------------------------------------------- */
+
+@import './bq-alert.variables';
+
+:host {
+  @apply block;
+}
+
+:host(.is-hidden) {
+  @apply hidden;
+}
+
+:host(.is-sticky) {
+  @apply fixed inset-0 top-0 z-[var(--bq-alert--z-index)] w-full;
+
+  .bq-alert {
+    @apply items-center justify-center rounded-none;
+  }
+}
+
+.bq-alert {
+  @apply relative flex min-w-[var(--bq-alert--min-width)] p-[var(--bq-alert--padding)];
+  @apply rounded-[var(--bq-alert--border-radius)] border-[length:--bq-alert--border-width];
+  border-style: var(--bq-alert--border-style);
+}
```

For entrance animation we can add the following:

```suggestion
.bq-alert {
  @include animation-fade-in;

  @apply relative flex min-w-[var(--bq-alert--min-width)] p-[var(--bq-alert--padding)];
  @apply rounded-[var(--bq-alert--border-radius)] border-[length:--bq-alert--border-width];

  border-style: var(--bq-alert--border-style);
}
```

This `animation-fade-in` is a SCSS mixing that can be found in this file:
`packages/beeq/src/global/styles/mixins/_keyframes.scss`